rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "keyvi"
-version = "0.5.3"
+version = "0.5.4"
```

Comment:
this shouldn't be part of this PR. 

We bump versions as part of a release, there are some more things to do for that (branching, tagging, ...)

Having that said, we can create a patch release soonish if you need the fix.

Reply:
thank you, removed version bump in 5dec183